projects/Allwinner/devices/H3/patches/u-boot/0007-sunxi-psci-Add-support-for-H3-CPU-0-hotplug.patch

```diff
@@ -182,7 +191,7 @@ index 563635636624..2f0d69bdfce2 100644
  #include <asm/arch/cpu.h>
  #include <linux/stringify.h>
  
-+#ifdef SUNXI_SRAM_A2_SIZE
```

Comment:
Need some help on correctly rebasing this line to support u-boot 2024.01. Otherwise it is probably good to be run tested / merged.

Reply:
I guess AW patches should be synched with https://github.com/smaeul/u-boot/commits/patch/h3-scp/